### PR TITLE
New rule: youtube-banner-show-upcoming-events

### PR DESCRIPTION
**Pull request by sethdailyssw**

### CODE REVIEW

File: rules/youtube-banner-show-upcoming-events/rule.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+type: rule
+archivedreason: 
+title: Does your home page banner show upcoming events?
+guid: 61d15bb1-aa8c-406c-97ea-4cd08bd22340
+uri: youtube-banner-show-upcoming-events
+created: 2023-08-11T03:56:22.0000000Z
+authors:
+- title: Seth Daily
```
Comment (camillars) [inline rules/youtube-banner-show-upcoming-events/rule.md:9]: Author should be @jontygardner as it was his suggestion

File: rules/youtube-banner-show-upcoming-events/rule.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  url: https://ssw.com.au/people/seth-daily
+related: 
+- promotion-do-people-know-about-your-event
+---
+
+Imagine a potential client visiting your YouTube channel, eager to engage with your content but unaware of the next big event you're hosting. By not highlighting upcoming events on your banner, you might miss a valuable opportunity to connect with your audience. You never want to miss an opportunity to show the events that you are hosting or sponsoring.
```
Comment (camillars) [inline rules/youtube-banner-show-upcoming-events/rule.md:15]: change from
> Imagine a potential client visiting your YouTube channel, eager to engage with your content but unaware of the next big event you're hosting. By not highlighting upcoming events on your banner, you might miss a valuable opportunity to connect with your audience. You never want to miss an opportunity to show the events that you are hosting or sponsoring.
to
Imagine a potential client visiting your YouTube channel, keen to engage with your content but clueless about the next event you're hosting. You skip a valuable chance to connect with your audience if you don't highlight upcoming events on your banner. It's a great way to show off what you're promoting or hosting!

**Review by camillars — CHANGES_REQUESTED** [review]

File: rules/youtube-banner-show-upcoming-events/rule.md (hunk 1 above)
Comment (sethdailyssw) [reply rules/youtube-banner-show-upcoming-events/rule.md]: done

File: rules/youtube-banner-show-upcoming-events/rule.md (hunk 2 above)
Comment (sethdailyssw) [reply rules/youtube-banner-show-upcoming-events/rule.md]: done

**Review by camillars — APPROVED** [review]

**Review by Geordie88 — APPROVED** [review]
Looks good

**Pull request merged**